cmd/commands/service.go

```diff
@@ -446,6 +451,34 @@ func ServiceDelete(fcClient *core.Client) *cobra.Command {
 	return command
 }
 
+func ServiceUnsubscribe(fcClient *core.Client) *cobra.Command {
+	deleteSubscriptionOptions := core.DeleteSubscriptionOptions{}
+
+	command := &cobra.Command{
+		Use:     "unsubscribe",
+		Short:   "Unsubscribe a service from an existing subscription",
+		Example: "  riff service unsubscribe my_subscription --namespace joseph-ns",
```

Comment:
Can't have underscores in k8s resource names

Reply:
Right! Let me fix that!